runtime/rasdump/trigger.c

```diff
@@ -1002,7 +1002,46 @@ triggerDumpAgents(struct J9JavaVM *vm, struct J9VMThread *self, UDATA eventFlags
 								OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
 								char dateStamp[64];
 								omrstr_ftime_ex(dateStamp, sizeof(dateStamp), "%Y/%m/%d %H:%M:%S", now, OMRSTR_FTIME_FLAG_LOCAL);
-								j9nls_printf(PORTLIB, J9NLS_INFO | J9NLS_STDERR | J9NLS_VITAL, J9NLS_DMP_PROCESSING_EVENT_TIME, mapDumpEvent(eventFlags), detailLength, detailData, dateStamp);
+
+								/* If there are more details about the event, print them.
+								 * During abort event detailData is empty string - skip the abort event.
+								 */
+								if (('\0' != *detailData) && (NULL != eventData->exceptionRef) && (NULL != *eventData->exceptionRef)) {
+									j9object_t emessage = J9VMJAVALANGTHROWABLE_DETAILMESSAGE(self, *eventData->exceptionRef);
+									if (NULL != emessage) {
+										UDATA subLength = strlen(detailData) + strlen("\" \"");
+										char *subDetailedMessage = (char *)j9mem_allocate_memory(subLength + 1, OMRMEM_CATEGORY_VM);
+										if (NULL != subDetailedMessage) {
+											char stackBuffer[256];
+											char *buf = stackBuffer;
+											UDATA len = 0;
+											strcpy(subDetailedMessage, detailData);
+											strcat(subDetailedMessage, "\" \"");
+											buf = self->javaVM->internalVMFunctions->copyStringToUTF8WithMemAlloc(self, emessage,
+													J9_STR_NULL_TERMINATE_RESULT, subDetailedMessage, subLength, stackBuffer,
+													sizeof(stackBuffer), &len);
+											j9mem_free_memory(subDetailedMessage);
+											if (NULL != buf) {
+												j9nls_printf(PORTLIB, J9NLS_INFO | J9NLS_STDERR | J9NLS_VITAL,
+														J9NLS_DMP_PROCESSING_EVENT_TIME, mapDumpEvent(eventFlags),
```

Comment:
`J9NLS_DMP_PROCESSING_EVENT_TIME` doesn't put double-quotes around the message in all translations; I think we should instead introduce a new NLS message which expects the additional exception message, which would also eliminate the need to allocate memory here and concatenate the pieces.